### PR TITLE
docs: update android integration guide

### PR DESCRIPTION
**Pull request by xiaoyijun**
update android integration guide

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:54]: - receive → 接收?
- endpoint → 端点
- . → 。
```suggestion
Redirect URI 指定了应用用来接收授权结果的端点。
```

File: docs/docs/recipes/integrate-logto/android.mdx
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/android.mdx:54]: Improved by Grammarly
```suggestion
The `Redirect URI` indicates the application's endpoint to receive authorization results.
```

File: docs/docs/recipes/integrate-logto/android.mdx
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/android.mdx:2]: maybe better keep ` (Kotlin)`?

Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:2]: Why? Java developers can also use this documentation.

**Review by IceHe — APPROVED** [review]
LGTM. Better wait for another approval.

File: docs/docs/recipes/integrate-logto/android.mdx
Comment (gao-sun) [inline docs/docs/recipes/integrate-logto/android.mdx:52]: you need put an admonition here like other recipes

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:54]: we don't need to translate endpoint

File: docs/docs/recipes/integrate-logto/android.mdx
Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:53]: updated.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx
Comment (xiaoyijun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx]: updated.

File: docs/docs/recipes/integrate-logto/android.mdx
Comment (gao-sun) [reply docs/docs/recipes/integrate-logto/android.mdx:53]: well i was think about the import of `<AppNote />` and `<SignInNote />` etc. would you like to go through the code and follow the latest integration guide format?

Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:58]: updated.

File: docs/docs/recipes/integrate-logto/android.mdx
Comment (demonzoo) [inline docs/docs/recipes/integrate-logto/android.mdx:14]: Fix these lint issues and we are good to move on

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx
Comment (demonzoo) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:227]: 退出登录

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx
Comment (demonzoo) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:229]: 执行退出登录？

File: docs/docs/recipes/integrate-logto/android.mdx
Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:13]: fixed.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx
Comment (xiaoyijun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx]: updated.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx
Comment (xiaoyijun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx]: removed.

**Review by demonzoo — APPROVED** [review]
LGTM

**Pull request merged**
